src/Translation/TranslationDelegate.php

```diff
@@ -15,6 +16,35 @@ public function __construct(TranslationApiInterface ...$apis)
     $this->apis = $apis;
   }
 
+  /**
+   * @throws InvalidArgumentException
+   */
+  public function translateProject(Program $project, ?string $source_language, string $target_language): ?array
+  {
+    $this->validateLanguage($source_language);
+    $this->validateLanguage($target_language);
+
+    $to_translate = [$project->getName(), $project->getDescription(), $project->getCredits()];
+    $translation_result = [];
+
+    foreach ($to_translate as $text) {
```

Comment:
Not trying too hard to merge the api calls into one. Couldn't find a way to insert special characters in between title, description, credit.
If one call fails, then all fail (return null).